### PR TITLE
2977: Fix schema for xml:lang attribute

### PR DESCRIPTION
**Pull request by rzymek01**
Related issue: #2977

### CODE REVIEW

**Comment by rzymek01** [comment]
This is a simple fix. Please merge it since all checks have passed.

**Comment by gtanczyk** [comment]
:thumbsup:

**Comment by rzymek01** [comment]
@spocke Is there anything that blocks you from merging this?

File: js/tinymce/classes/html/Schema.js [hunk 1]
```diff
@@ -243,8 +243,8 @@ define("tinymce/html/Schema", [
 
 		// Extend with HTML4 attributes unless it's html5-strict
 		if (type != "html5-strict") {
-			addAttrs("script", "language xml:space");
-			addAttrs("style", "xml:space");
+			addAttrs("script", "language xml::space");
+			addAttrs("style", "xml::space");
```
Comment (jayarjo) [inline js/tinymce/classes/html/Schema.js]: Why two colons here?

**Comment by jayarjo** [comment]
Thank you for suggestion, but we couldn't merge it the way you proposed it. So I've added a way to escape colons in the attribute names:

```js
...
valid_elements: 'pre[xml\\:lang]'
...
```

**Comment by rzymek01** [comment]
@jayarjo I see, thanks for fixing this! 👍

**Pull request closed without merging**